exonum/src/messages/protocol.rs

```diff
@@ -361,3 +362,15 @@ messages! {
         height: Height,
     }
 }
+
+impl BlockResponse {
+    /// Verify Merkle root of transactions in the block.
+    pub fn verify_tx_hash(&self) -> bool {
+        let db = MemoryDB::new();
```

Comment:
I'm a little bit concerned about creating `MemoryDB` here, but probably it is OK.

Reply:
@SerhiiOryshych  can we create task about removing `MemoryDb`?